compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/capturing-function-alias-computed-load-3-iife.expect.md

```diff
@@ -37,8 +37,6 @@ function bar(a, b) {
     let t;
     t = {};
 
-    y;
```

Comment:
I believe this came from the artificial / fake LoadLocal (created for `LoweredFunction.dependencies`)